tests/py/dynamo/testing_utilities.py

```diff
@@ -14,6 +14,7 @@
 )
 
 DECIMALS_OF_AGREEMENT = 4
+DECIMALS_OF_AGREEMENT_3 = 3
```

Comment:
Is this needed due to a change in this PR?

Reply:
reverted this change. The reason for this change was test_lower_linear wasn't meeting threshold requirement. Reverted to check the latest status on CI.